CHANGELOG.md

```diff
@@ -60,6 +60,23 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
   `HEADER_LENGTH` remains the same, first byte of `RawMessage` is now reserved and
   always set to `0`. (#579)
 
+- `exonum::explorer` module has been reworked to add new functionality. (#535)
+  In particular:
+
+  - The explorer now allows to iterate over blocks in the blockchain
+    in the given height range, replacing old `blocks_range` method
+  - `block_info` and `tx_info` methods of the explorer are renamed to
+    `block` and `transaction`, respectively
+  - `TransactionInfo` moved from the `api::public` module to the `explorer` module
+  - `BlocksRange` moved from the `explorer` module to the `api::public` module
+  - `TxInfo` is renamed to `CommittedTransaction`
+  - `BlockInfo` has its fields made private, instead providing getter methods
```

Comment:
The last sentence sounds complicated, imo. What if we just say?
- `BlockInfo` fields are private now, yet accessible with getter methods

Reply:
Btw, as our changelog list items are complete sentences we should use periods at the end (as we already did :) )

Reply:
I agree with @mvlabat (especially with the comment about dots), but I want to merge this pull request already, so we will deal with this issue in a separate pull request. :pray: